bin/run_dexseq.r

```diff
@@ -139,4 +148,4 @@ suppressWarnings({
   dex.padj <- getAdjustedPValues(stageRObj, order=FALSE,
                                  onlySignificantGenes=FALSE)
 })
-write.csv(cbind(dxr_pval,dex.padj$gene,dex.padj$transcript), file="dexseq.results.txt")
+write.csv(cbind(dxr_pval,dex.padj$gene,dex.padj$transcript), file="DEXseq_out.txt")
```

Comment:
```suggestion
write.csv(cbind(dxr_pval,dex.padj$gene,dex.padj$transcript), file="dexseq.results.txt")
```

Reply:
Better to have it named like this. I have also documented it like this in `output.md`

Reply:
Hi @drpatelh, we have changed it to "dexseq.results.txt".